torch/fx/graph_module.py

```diff
@@ -138,6 +141,7 @@ def __init__(self, root: Union[torch.nn.Module, Dict[str, Any]], graph: Graph):
         if isinstance(root, torch.nn.Module):
             if hasattr(root, 'training'):
                 self.training = root.training
+            self._buffers = root._buffers
```

Comment:
I think buffers should be handled now after https://github.com/pytorch/pytorch/pull/47641, at least ones referenced in the FX graph